internal/domain/http.go

```diff
@@ -3,35 +3,104 @@
 
 package domain
 
+import (
+	"fmt"
+)
+
+type StatusCode int
+
 const (
-	StatusNoMatches                = 200
-	StatusResolutionMismatch       = 201
-	StatusSourceMismatch           = 202
-	StatusRlsGrpMismatch           = 203
-	StatusCutMismatch              = 204
-	StatusEditionMismatch          = 205
-	StatusRepackStatusMismatch     = 206
-	StatusHdrMismatch              = 207
-	StatusStreamingServiceMismatch = 208
-	StatusAlreadyInClient          = 210
-	StatusNotASeasonPack           = 211
-	StatusBelowThreshold           = 230
-	StatusSuccessfulMatch          = 250
-	StatusSuccessfulHardlink       = 250
-	StatusFailedHardlink           = 440
-	StatusClientNotFound           = 472
-	StatusGetClientError           = 471
-	StatusDecodingError            = 470
-	StatusAnnounceNameError        = 469
-	StatusGetTorrentsError         = 468
-	StatusTorrentBytesError        = 467
-	StatusDecodeTorrentBytesError  = 466
-	StatusParseTorrentInfoError    = 465
-	StatusGetEpisodesError         = 464
-	StatusEpisodeCountError        = 450
+	StatusNoMatches                StatusCode = 200
+	StatusResolutionMismatch       StatusCode = 201
+	StatusSourceMismatch           StatusCode = 202
+	StatusRlsGrpMismatch           StatusCode = 203
+	StatusCutMismatch              StatusCode = 204
+	StatusEditionMismatch          StatusCode = 205
+	StatusRepackStatusMismatch     StatusCode = 206
+	StatusHdrMismatch              StatusCode = 207
+	StatusStreamingServiceMismatch StatusCode = 208
+	StatusAlreadyInClient          StatusCode = 210
+	StatusNotASeasonPack           StatusCode = 211
+	StatusBelowThreshold           StatusCode = 230
+	StatusSuccessfulMatch          StatusCode = 250
+	StatusSuccessfulHardlink       StatusCode = 250
+	StatusFailedHardlink           StatusCode = 440
+	StatusClientNotFound           StatusCode = 472
+	StatusGetClientError           StatusCode = 471
+	StatusDecodingError            StatusCode = 470
+	StatusAnnounceNameError        StatusCode = 469
+	StatusGetTorrentsError         StatusCode = 468
+	StatusTorrentBytesError        StatusCode = 467
+	StatusDecodeTorrentBytesError  StatusCode = 466
+	StatusParseTorrentInfoError    StatusCode = 465
+	StatusGetEpisodesError         StatusCode = 464
+	StatusEpisodeCountError        StatusCode = 450
 )
 
-var StatusMap = map[string][]int{
+func (s StatusCode) Message() string {
```

Comment:
If this is changed to `String() string` it will satisfy the Stringer interface so it can be used in `fmt.Printf("msg: %s", code)` for example.